src/locales/en/config.json

```diff
@@ -1,8 +1,8 @@
 {
 	"habits": {
-		"example1": "Exercise",
-		"example2": "Book",
-		"example3": "Hobby",
```

Comment:
It seems like you translated the wrong file

Reply:
Jup!

Reply:
you should probably revert the files you accidentally changed, s.t. the blame history doesn't get poluted.